src/Microsoft.AspNetCore.Authorization/AuthorizationOptions.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.AspNetCore.Authorization
     /// </summary>
     public class AuthorizationOptions
     {
-        private IDictionary<string, AuthorizationPolicy> PolicyMap { get; } = new Dictionary<string, AuthorizationPolicy>(StringComparer.OrdinalIgnoreCase);
```

Comment:
I guess we should make it a method call then, GetPolicyMap() to try and signal that its a read only copy